common/src/main/java/com/wynntils/models/gearinfo/parsing/GearParseResult.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gearinfo.parsing;
+
+import com.wynntils.models.concepts.Powder;
+import com.wynntils.models.gearinfo.type.GearTier;
+import com.wynntils.models.gearinfo.type.GearType;
+import com.wynntils.models.stats.type.StatActualValue;
+import java.util.List;
+
+// "tierCount" depends on what item were parsed. For identified items, it is the
+// number of rerolls. For crafted items, it is the current durability
```

Comment:
Bit weird.. But okay, it's fine. I assume this is just some internal helper class.

Reply:
Yes, I agree. It is a bit weird. It used to be just `rerollCount` until I started supporting crafted gear. 

You are right, this is just an internal class bundling the results of the gear parsing. 

An alternative is of course to change this to have like both "rerollCount" and a "currentDurability", and only ever set one of them. Maybe that is clearer?

Reply:
Second is a bit cleaner but it really doesn't matter

Reply:
Fixed.